### ISSUE TITLE
Post-installation Messages - Open links in new tab

### ISSUE THREAD

**Issue by BurtNL** [joomla/joomla-cms#22061]
### Steps to reproduce the issue
View the Post-Installation Messages and click some of the links which refers to other information.
Some links opens in a new tab, some links opens in the Admin itself.
Preferred: open all links to more information in a new tab.

### Expected result
Open the links to more information in a new tab.

### Actual result
Some links opens in new tab, some don't.

### System information (as much as possible)
Joomla 3.9.0-alpha2-dev, but might occur in previous versions as well.

### Additional comments
Should be fixed in the language files.

**Comment by ghost** [comment]
>  Preferred: open all links to more information in a new tab.

my personal favorite too. But i learned that open in a new ~~Windows~~ Tab isn't user-friendly as it should belong to the User if open in a new ~~Window~~ Tab or not.

**Comment by Quy** [comment]
Let's make a final decision to either open in a new tab or same tab so it can be changed accordingly.

**Comment by ghost** [comment]
@Quy For Accessibility-Reasons it should be the same Tab.

**Comment by brianteeman** [comment]
This should be closed as the current behaviour is correct

**Comment by ghost** [comment]
closed as stated above.
